doc/src/pair_mbx.rst

```diff
@@ -0,0 +1,203 @@
+.. index:: pair_style mbx
+
+pair_style mbx command
+======================
+
+Syntax
+""""""
+
+.. code-block:: LAMMPS
+
+    pair_style mbx cutoff
+
+* cutoff = real-space cutoff for MBX in Angstroms
+
+
+Examples
+""""""""
+
+.. code-block:: LAMMPS
+
+    # For a system involving water (atom types O=1, H=2)
+    processors      * * * map xyz
```

Comment:
The processors keyword should not be needed to demonstrate the usage of the pair style.
Same applies to the compute command. Also the comments should be removed here and we don't as many examples here. This part of the documentation should just show the syntax, so one plain example and one with hybrid should suffice. The explanations in the comments belong to the example input decks to which there is a reference given below.